api/bazel/repository_locations.bzl

```diff
@@ -100,9 +100,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "OpenTelemetry Proto",
         project_desc = "Language Independent Interface Types For OpenTelemetry",
         project_url = "https://github.com/open-telemetry/opentelemetry-proto",
-        version = "0.11.0",
-        sha256 = "985367f8905e91018e636cbf0d83ab3f834b665c4f5899a27d10cae9657710e2",
-        release_date = "2021-10-07",
+        version = "0.15.0",
```

Comment:
version `0.16.0` is now released

Reply:
I'm getting some good practice at updating the OTLP proto version :) Updated, thanks!